modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -50,6 +50,11 @@ system using available RHEL tools, such as `nmcli` or `nmtui`.
 $ coreos-installer install --copy-network \
      --ignition-url=http://host/worker.ign /dev/sda
 ----
++
+[IMPORTANT]
+====
+The `--copy-network` option will only copy configuration files found under `/etc/NetworkManager/system-connections`.
```

Comment:
What about something like:

```suggestion
The `--copy-network` option only copies networking configuration found under `/etc/NetworkManager/system-connections`. In particular, it does not copy the system hostname.
```

That could help users who don't know where the hostname lives.